test/shm_size.bats

```diff
@@ -5,14 +5,27 @@ load helpers
 function setup() {
 	setup_test
 	# Start it with the environment variable set, don't use replace_config
-	CONTAINER_ENABLE_CUSTOM_SHM_SIZE=true start_crio
 }
 
 function teardown() {
 	cleanup_test
 }
 
+function create_shmsize_runtime() {
+	cat << EOF > "$CRIO_CONFIG_DIR/01-shmsize.conf"
+[crio.runtime]
+default_runtime = "shmsize"
+[crio.runtime.runtimes.shmsize]
+runtime_path = "$RUNTIME_BINARY"
+runtime_root = "$RUNTIME_ROOT"
+runtime_type = "$RUNTIME_TYPE"
+allowed_annotations = ["io.kubernetes.cri-o.ShmSize"]
```

Comment:
Can `allowed_annotations` be specified on a global (rather than per-runtime) basis? The way it is configured now kinda overloads the meaning of a runtime.

Reply:
not currently!